site/_includes/partials/chrome-100-card-section.njk

```diff
@@ -0,0 +1,2469 @@
+          <div class="display-flex gap-top-200 logos">
+            <img src="/images/project/promise-tag.svg" alt="Promise tag" class="tags gap-top-100">
+            <div class="display-flex likes">
+              <a class="heart-logo" data-id="4" data-year="2008">
+                <svg width="20" height="19" viewBox="0 0 20 19" fill="none" xmlns="http://www.w3.org/2000/svg">
+                  <path d="M8.88659 16.6603L8.88587 16.6596C6.30081 14.3155 4.19567 12.4057 2.73078 10.6147C1.27162 8.83074 0.5 7.22576 0.5 5.5C0.5 2.69614 2.69614 0.5 5.5 0.5C7.08861 0.5 8.62112 1.24197 9.61932 2.41417L10 2.8612L10.3807 2.41417C11.3789 1.24197 12.9114 0.5 14.5 0.5C17.3039 0.5 19.5 2.69614 19.5 5.5C19.5 7.22577 18.7284 8.83077 17.2691 10.6161C15.8065 12.4055 13.7058 14.3144 11.1265 16.6584L11.1148 16.669L11.1137 16.67L10.0013 17.675L8.88659 16.6603Z" stroke="#3C4043" class="fill-like">
+                </svg>
+              </a>
+              <span class="like-count"></span>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="card gap-bottom-300">
+        <div class="card-info card-top">
+          <div class="display-flex arrow-wrapper">
+            <a href="https://webkit.org/blog/280/full-pass-of-acid-3/" target="_blank" rel="noopener noreferrer">
+              <h1 class="card-heading">WebKit fully passes ACID3</h1>
+            </a>
+            <img src="/images/project/arrow.svg" alt="Arrow" class="right-arrow">
+          </div>
+          <p class="card-description">Third condition met. All power ahead.</p>
+          <div class="display-flex gap-top-200 logos">
+            <img src="/images/project/promise-tag.svg" alt="Promise tag" class="tags gap-top-100">
+            <div class="display-flex likes">
+              <a class="heart-logo" data-id="5" data-year="2008">
+                <svg width="20" height="19" viewBox="0 0 20 19" fill="none" xmlns="http://www.w3.org/2000/svg">
+                  <path d="M8.88659 16.6603L8.88587 16.6596C6.30081 14.3155 4.19567 12.4057 2.73078 10.6147C1.27162 8.83074 0.5 7.22576 0.5 5.5C0.5 2.69614 2.69614 0.5 5.5 0.5C7.08861 0.5 8.62112 1.24197 9.61932 2.41417L10 2.8612L10.3807 2.41417C11.3789 1.24197 12.9114 0.5 14.5 0.5C17.3039 0.5 19.5 2.69614 19.5 5.5C19.5 7.22577 18.7284 8.83077 17.2691 10.6161C15.8065 12.4055 13.7058 14.3144 11.1265 16.6584L11.1148 16.669L11.1137 16.67L10.0013 17.675L8.88659 16.6603Z" stroke="#3C4043" class="fill-like">
+                </svg>
+              </a>
+              <span class="like-count"></span>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  {# Year 2009 #}
+  <section class="inner-card display-flex center-margin gap-right-300 year-2009" id="section2009" data-index="1">
+    <div class="first-column gap-right-300">
+      <div class="card card-with-img gap-bottom-300">
+        <img src="/images/meta/2009.png" alt="Placeholder" class="card-img">
```

Comment:
As mentioned elsewhere, all of the images will need to be run through our Image CDN.

That would potentially allow you to use the `Img` shortcode, which supports lazy-loading by default. If, for some reason, you can't use the `Img` shortcode, please consider adding in [lazy-loading as an attribute](https://developer.mozilla.org/en-US/docs/Web/Performance/Lazy_loading) instead.

Reply:
Done.